src/main/java/edu/ksu/canvas/attendance/entity/AttendanceAssignment.java

```diff
@@ -0,0 +1,137 @@
+package edu.ksu.canvas.attendance.entity;
+
+
+import javax.persistence.*;
+import java.io.Serializable;
+
+@Entity
+@Table(name = "attendance_assignment")
+public class AttendanceAssignment implements Serializable {
+
+    private static final long serialVersionUID = 1L;
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    @Column(name = "assignment_id")
+    private Long assignmentId;
+
+    @OneToOne(cascade = CascadeType.ALL)
+    @JoinColumn(name = "section_id", foreignKey = @ForeignKey(name = "fk_section_attendance"), nullable = false)
+    private AttendanceSection attendanceSection;
+
+    @Column(name = "canvas_assignment_id")
+    private Long canvasAssignmentId;
+
+    @Column(name = "assignment_points")
+    private Double assignmentPoints;
+
+    @Column(name = "present_points")
+    private Double presentPoints;
+
+    @Column(name = "tardy_points")
+    private Double tardyPoints;
+
+    @Column(name = "excused_points")
+    private Double excusedPoints;
+
+    @Column(name = "absent_points")
+    private Double absentPoints;
+
+    @Column(name = "grading_on", nullable = false)
+    private Boolean gradingOn = false;
```

Comment:
Since this is always set and not nullable you might want to use a primitive instead of the boxed boolean